pom.xml

```diff
@@ -192,14 +192,6 @@
 				<version>3.1</version>
 				<scope>test</scope>
 			</dependency>
-
-			<dependency>
-				<groupId>org.hectorclient</groupId>
-				<artifactId>hector-core</artifactId>
-				<version>1.1-4</version>
-				<scope>test</scope>
-			</dependency>
```

Comment:
I removed this dependency because it's not used by the code as far as I can tell and the project itself is abandoned.

Maybe this can be moved to a separate PR, lemme know.